lighthouse-core/test/runner-test.js

```diff
@@ -670,31 +670,6 @@ describe('Runner', () => {
     });
   });
 
-  it('includes any LighthouseRunWarnings from errored audits in LHR', () => {
```

Comment:
this is the sole loss of functionality (being able to push top-level warnings before the audit completely errors out), but I don't think this has ever been behavior anyone cares about or needed

Reply:
maybe call out the loss of this as a breaking change in PR description so we remember to include that too in final notes?

Reply:
> maybe call out the loss of this as a breaking change in PR description so we remember to include that too in final notes?

done